test/fixtures/userStatus/userStatus.js

```diff
@@ -0,0 +1,56 @@
+const userStsDataForOooState = (userId) => {
+  return {
+    userId,
+    currentStatus: {
+      until: "1669256009000",
+      message: "Bad Health",
+      state: "OOO",
+      updatedAt: "1668215609000",
+      from: "1668215609000",
+    },
+    monthlyHours: {
+      updatedAt: "1668215609000",
+      committed: "40",
+    },
+  };
+};
+
+const invalidUserStsDataforPost = (userId) => {
+  return {
+    userId,
+    currentStatus: {
+      until: "",
+      message: "",
+      state: "ACTIVE",
+      updatedAt: "1668215609000",
+      from: "1668215609000",
```

Comment:
Why are the tests passing? I though `joi` was enforcing `number` type for `from`. Here it is the incorrect string type.

Reply:
The tests were passing because Joi converts strings if it can by default. However, if we want it to be a strict number we can add `.strict()` in the validation.